app/models/relationship.rb

```diff
@@ -1,4 +1,7 @@
 class Relationship < ApplicationRecord
+  scope :pending, -> { where(status: :pending) }
```

Comment:
You don't need these scopes, enum creates them by default

Reply:
Fixed